### ISSUE TITLE
DatePicker/ DateRangePicker date component with write property

### ISSUE THREAD

**Issue by lucasjamar** [snehilvj/dash-mantine-components#23]
Hi! Thanks for the awesome lib! Its really nice to finally have a date picker that can go back in time quickly. I even mentioned this in  `dcc`: https://github.com/plotly/dash-core-components/issues/990
However, the `date`/`dates` properties of the date pickers appear to be read only. It would be great to add write properties to them so that in a callback, you can for example set a default date range. Example code:

```
from datetime import datetime, timedelta
from dash import Dash, html, Input, Output
import dash_mantine_components as dmc

# App
app = Dash(__name__)
server = app.server

app.layout = html.Div(
    children=[
        html.Button("Set dates", id="click"),
        dmc.DateRangePicker(
            id="date_range",
        )
    ]
)


@app.callback(
    Output("date_range", "dates"),
    Input("click", "n_clicks"),
)
def set_initial_dates(click):
    today = datetime.now()
    end_date = today + timedelta(10)
    return [today, end_date]


# Main
if __name__ == "__main__":
    app.run_server(debug=True)
```

![date_write](https://user-images.githubusercontent.com/39372418/148782105-ec9cb87e-62a7-4682-8d32-57e4b9a519a1.gif)

**Comment by snehilvj** [comment]
Hi @lucasjamar thanks for creating this issue. This is an important feature, I'll work on this on priority.

**Comment by snehilvj** [comment]
This has been fixed, will be part of the next release: 0.2.3.
